### PR TITLE
vulnerabilities hotfix: cryptiles and jsonparser

### PR DESCRIPTION
**Pull request by mingzhangyang**
Updated the dev-dependencies that are dependent on cryptiles and jsonparson packages

### CODE REVIEW

**Comment by lanczyck** [comment]
I don't see cryptiles in the new package-lock.json at all.  What was it replaced by?  bcrypt-pbkdf?

**Comment by lanczyck** [comment]
Also, there are an enormous number of diffs in package-lock.json.  Other than the cryptiles issue, do you know which diffs are specific to the two APPSEC jira tickets?

**Comment by mingzhangyang** [comment]
HI Chris, I uninstall the dev-dependent packages that requires cryptiles, and reinstall its latest version. Then cryptiles is no longer required in the latest version. Usually, we don't touch package-lock.json, it will be regenerated automatically after changes in package.json are made. I have run `grunt` command, and it passes the tests.

File: package-lock.json [hunk 1]
```diff
@@ -1650,105 +2208,75 @@
...
-          "integrity": "sha1-riXPJRKziFodldfwN4aNhDESR2U=",
-          "dev": true
         }
       }
     },
     "engine.io-client": {
-      "version": "1.8.5",
-      "resolved": "https://registry.npmjs.org/engine.io-client/-/engine.io-client-1.8.5.tgz",
-      "integrity": "sha512-AYTgHyeVUPitsseqjoedjhYJapNVoSPShbZ+tEUX9/73jgZ/Z3sUlJf9oYgdEBBdVhupUpUqSxH0kBCXlQnmZg==",
+      "version": "3.2.1",
```
Comment (lanczyck) [inline package-lock.json:2240]: APPSEC-44 says engine.io-client v3.2.2 avoids the parsejson dependency entirely.  Was there a reason to stay with 3.2.1?

**Pull request merged**